README.md

```diff
@@ -453,7 +453,7 @@ Used by FastAPI / Starlette:
 * <a href="https://www.uvicorn.org" target="_blank"><code>uvicorn</code></a> - for the server that loads and serves your application.
 * <a href="https://github.com/ijl/orjson" target="_blank"><code>orjson</code></a> - Required if you want to use `ORJSONResponse`.
 
-You can install all of these with `pip install fastapi[all]`.
+You can install all of these with `pip install fastapi[all]`. If you need a subset of dependencies from this list, you can also request them by name with `pip install fastapi[request,aiofiles,pyyaml]`. This will ensure that dependence installed will conform to the rages FastAPI expects.
```

Comment:
Typo in the last sentence: `This will ensure that dependence installed will conform to the rages FastAPI expects.` -> `This will ensure that the dependencies installed with conform to the ranges FastAPI expects`.

Also, I think you want to replace `request` with `requests` in the `pip install` example.